epic/check_autobranch.go

```diff
@@ -183,6 +183,11 @@ func mergeSucceedItem(
 		return true
 	}
 
+	if info.Status == "neutral" {
```

Comment:
I just knew [`conclusion`](https://docs.github.com/en/rest/reference/checks#create-a-check-run) field now....

I think we should define `StateChangeInfo.Conclusion` and check it at here.

By #261, we fill `StateChangeInfo.Status` in two way:

- https://github.com/voyagegroup/popuko/blob/c1901c2b115bbd84e213e83d029df0e3c78b392e/epic/check_autobranch.go#L27
- https://github.com/voyagegroup/popuko/blob/c1901c2b115bbd84e213e83d029df0e3c78b392e/epic/check_autobranch.go#L41

But `github.StatusEvent.Status` and `github.StatusEvent.Conclusion` are different field. It's time to stop to mix them into single field. Let's handle them separately.

Reply:
> `github.StatusEvent.Status` and `github.StatusEvent.Conclusion` are different field. It's time to stop to mix them into single field.

Yes, from code design point of view, that's a good idea.
However, some refactoring is considered necessary.

BTY, I'm sorry to postpone the issue, but how about the following changes.

```diff
- NotHandle                 bool
+ InProgress                 bool
```

```diff
- NotHandle:                 *ev.CheckSuite.Status != "completed",
+ InProgress:                 *ev.CheckSuite.Status != "completed" || *ev.CheckSuite.Conclusion == "neutral"
```

Reply:
@yamachu 

Even though, I suspect it's better to add `StateChangeInfo.InProgress()` method instead of your proposed approach.
It's nice abstraction to push something into the appropriate method. I think that changing struct field as your proposed is not robustness for the future. 

Then we would need to change to hold values of `github.StatusEvent.Status` and `github.StatusEvent.Conclusion` in `StateChangeInfo`. This requires a similar refactoring as I told the above.